app/javascript/shared/components/ui/label/LabelDropdown.vue

```diff
@@ -97,6 +102,41 @@ export default {
         this.onAdd(label);
       }
     },
+    onArrowUp() {
+      const array = this.$refs.labelList.querySelectorAll(
+        'div > ul > li > div > button'
```

Comment:
Can we use classnames to target the button, instead using `ul`, `li`  directly ?